src/dsp_tools/clients/ontology_client_live.py

```diff
@@ -0,0 +1,112 @@
+from dataclasses import dataclass
+from typing import Any
+
+import requests
+from loguru import logger
+from rdflib import Graph
+from rdflib import Literal
+from rdflib import URIRef
+from requests import ReadTimeout
+from requests import Response
+
+from dsp_tools.clients.authentication_client import AuthenticationClient
+from dsp_tools.clients.ontology_client import OntologyClient
+from dsp_tools.error.exceptions import BadCredentialsError
+from dsp_tools.error.exceptions import BaseError
+from dsp_tools.error.exceptions import InternalError
+from dsp_tools.utils.rdflib_constants import KNORA_API
+from dsp_tools.utils.rdflib_utils import serialise_json
+from dsp_tools.utils.request_utils import RequestParameters
+from dsp_tools.utils.request_utils import log_and_raise_timeouts
+from dsp_tools.utils.request_utils import log_request
+from dsp_tools.utils.request_utils import log_response
+
+TIMEOUT = 60
+
+HTTP_LACKING_PERMISSIONS = 403
```

Comment:
there are HTTP status codes built in to Python: https://docs.python.org/3/library/http.html#http.HTTPStatus